Automattic/wp-calypso
# TinyMCE: Remove unused Memberships code

#### Changes proposed in this Pull Request

* In the process of migrating `UpgradeNudge` to `UpsellNudge`, we found this code for a Memberships integration with TinyMCE (similar to that of Simple Payments) that does not appear to be used.
* This PR removes the unused code so we can proceed with removing `UpgradeNudge` (See #38778 )

#### Testing instructions

* Switch to this PR
* Make sure nothing breaks as a result of removing this code. Check on a site with the Classic editor enabled, as well as the block editor.
